docs/src/literate-tutorials/linear_elasticity.jl

```diff
@@ -1,3 +1,408 @@
+dh = DofHandler(grid)
+add!(dh, :u, ip)
+close!(dh);
+
+# ### Boundary conditions
+# We set Dirichlet boundary conditions by fixing the motion normal to the bottom and left
+# boundaries. The last argument to `Dirichlet` determines which components of the field should be
+# constrained. If no argument is given, all components are constrained by default.
+ch = ConstraintHandler(dh)
+add!(ch, Dirichlet(:u, getfacetset(grid, "bottom"), (x, t) -> 0.0, 2))
+add!(ch, Dirichlet(:u, getfacetset(grid, "left"),   (x, t) -> 0.0, 1))
+close!(ch);
+
+# In addition, we will use Neumann boundary conditions on the top surface, where
+# we add a traction vector of the form
+# ```math
+# \boldsymbol{t}_\mathrm{N}(\boldsymbol{x}) = (20e3) x_1 \boldsymbol{e}_2\ \mathrm{N}/\mathrm{mm}^2
+# ```
+traction(x) = Vec(0.0, 20e3 * x[1]);
+
+# On the right boundary, we don't do anything, resulting in a zero traction Neumann boundary.
+# In order to assemble the external forces, $f_i^\mathrm{ext}$, we need to iterate over all
+# facets in the relevant facetset. We do this by using the [`FacetIterator`](@ref).
+
+function assemble_external_forces!(f_ext, dh, facetset, facetvalues, prescribed_traction)
+    ## Create a temporary array for the facet's local contributions to the external force vector
+    fe_ext = zeros(getnbasefunctions(facetvalues))
+    for face in FacetIterator(dh, facetset)
+        ## Update the facetvalues to the correct facet number
+        reinit!(facetvalues, face)
+        ## Reset the temporary array for the next facet
+        fill!(fe_ext, 0.0)
+        ## Access the cell's coordinates
+        cell_coordinates = getcoordinates(face)
+        for qp in 1:getnquadpoints(facetvalues)
+            ## Calculate the global coordinate of the quadrature point.
+            x = spatial_coordinate(facetvalues, qp, cell_coordinates)
+            tₚ = prescribed_traction(x)
+            ## Get the integration weight for the current quadrature point.
+            dΓ = getdetJdV(facetvalues, qp)
+            for i in 1:getnbasefunctions(facetvalues)
+                Nᵢ = shape_value(facetvalues, qp, i)
+                fe_ext[i] += tₚ ⋅ Nᵢ * dΓ
+            end
+        end
+        ## Add the local contributions to the correct indices in the global external force vector
+        assemble!(f_ext, celldofs(face), fe_ext)
+    end
+    return f_ext
+end
+#md nothing #hide
+
+# ### Material behavior
+# Next, we need to define the material behavior, specifically the elasticity tensor $\mathsf{C}$.
+# In this tutorial, we use plane strain linear isotropic elasticity, with Hooke's law as
+# ```math
+# \boldsymbol{\sigma} = 2G \boldsymbol{\varepsilon}^\mathrm{dev} + 3K \boldsymbol{\varepsilon}^\mathrm{vol}
+# ```
+# where $G$ is the shear modulus and $K$ the bulk modulus. The linearity of this expression implies that
+# can be represented by the general expression, $\boldsymbol{\sigma} = \mathsf{C}:\boldsymbol{\varepsilon}$,
```

Comment:
Sorry, I am not sure about the statement here. Maybe we can rephrase it a bit to be more precise here to not confuse user.

```suggestion
# where $G$ is the shear modulus and $K$ the bulk modulus. Note that the elasticity tensor can be expressed by the general expression, $\boldsymbol{\sigma} = \mathsf{C}:\boldsymbol{\varepsilon}$. We will use this form in the remaining tutorial.
```

Reply:
Changing to
```suggestion
# where $G$ is the shear modulus and $K$ the bulk modulus. This expression can be written as
# $\boldsymbol{\sigma} = \mathbb{C}:\boldsymbol{\varepsilon}$, with
```
hopefully this is easier.